### DISCUSSION TITLE
How to edit model from Google Poly in Spoke?

### DISCUSSION THREAD

**Discussion by davegoopot** [mozilla/hubs#2872] [General Questions & Discussion] [answered]
There is is a model of a theatre in Google Poly that is marked as remixable [1]. When I load this into Spoke I have the whole model, but I can't edit it. How can I change the model for use in Hubs? E.g for that theatre, how do I change the colour of the seats?

[1] https://poly.google.com/view/8jExwEzwPAe

**Comment by rawnsley** [comment, marked as answer]
Spoke is designed for arranging models not for editing them. You will need to edit the model in something like Blender before reimporting it into Spoke.
